ton_client/client/tonclient.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
```

Comment:
Why this file is added in the PR?

Reply:
It is a common c/c++ header for all bindings that will link a core library.